azure/functions/blob.py

```diff
@@ -1,31 +1,31 @@
 import io
-import typing
+from typing import Optional, Union, Any
```

Comment:
This is better, as we only import what's actually needed and make it explicit at the top. We should slowly be changing all the `import *` statements to `from * import **` statements.